### PR TITLE
Microsoft Management small changes

### PR DESCRIPTION
**Pull request by orlichter1**
##  Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26467

## Description
The format of startTime and endTime in the command is wrong.
Add Publisher Identifier to the HTTP calls in order to have better quota
Think of adding a timeout parameter to the get-content
Subscribing to an already subscribed channel returns an error. Fix it so it won't be an error only a message.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/MicrosoftManagementActivity.py [hunk 1]
```diff
@@ -155,17 +182,23 @@ def start_or_stop_subscription_request(self, content_type, start_or_stop_suffix)
...
-        response = self._http_request(
-            method='POST',
-            url_suffix=self.suffix_template.format(self.tenant_id, start_or_stop_suffix),
-            headers=headers,
-            params=params,
-            ok_codes=(200, 201, 202, 203, 204),
-            return_empty_response=True
-        )
-        return response
+        try:
```
Comment (yaakovi) [inline Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/MicrosoftManagementActivity.py:188]: why do you need this code in addition to the modification of the wrapper function?
why can't you leave this function as is and catch the error only in the wrapper function?
is this function being used somewhere else?

File: Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/MicrosoftManagementActivity.py [hunk 2]
```diff
@@ -277,13 +313,13 @@ def get_content_records_context(content_records):
...
 
 
-def get_all_content_type_records(client, content_type, start_time, end_time):
-    content_blobs = client.list_content_request(content_type, start_time, end_time)
+def get_all_content_type_records(client, content_type, start_time, end_time, timeout=10):
+    content_blobs = client.list_content_request(content_type, start_time, end_time, timeout)
     # The list_content request returns a list of content records, each containing a url that holds the actual data
     content_uris = [content_blob.get('contentUri') for content_blob in content_blobs]
-    content_records = []
+    content_records: List = list()
```
Comment (yaakovi) [inline Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/MicrosoftManagementActivity.py:320]: why did you need to change that?
`[]` is better than `list()`

File: Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/README.md [hunk 3]
```diff
@@ -197,7 +197,8 @@ Returns all content of a specific content type.
...
 | record_types_filter | A comma-separated list of the record types to fetch. Content records with a record  type that isn&#x27;t specified will not be fetched. If this field is left empty, all record types will be fetched. | Optional | 
 | workloads_filter | A comma-separated list of the workloads to fetch. Content records with a workload that isn&#x27;t specified will not be fetched. If this field is left empty, all workloads will be fetched. | Optional | 
-| operations_filter | A comma-separated list of the operations to fetch. Content records with an operation that isn&#x27;t specified will not be fetched. If this field is left empty, all operations will be fetched. | Optional | 
+| operations_filter | A comma-separated list of the operations to fetch. Content records with an operation that isn&#x27;t specified will not be fetched. If this field is left empty, all operations will be fetched. | Optional |
```
Comment (yaakovi) [inline Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/README.md:200]: update all the time formats here as well

**Review by yaakovi — APPROVED** [review]

File: Packs/MicrosoftManagementActivity/ReleaseNotes/1_1_0.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Microsoft Management Activity API (O365 Azure Events)
+- Added a `timeout` argument to **ms-management-activity-list-content** in order to fetch larger amounts of data.
+- Fixed a bug that occurred when resubscribing to a subscription. 
+- Updated docker image.
```
Comment (yaakovi) [inline Packs/MicrosoftManagementActivity/ReleaseNotes/1_1_0.md:6]:
```suggestion
- Updated docker image to demisto/pyjwt3:1.0.0.8871.
```

File: Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/MicrosoftManagementActivity.py (hunk 2 above)
Comment (orlichter1) [reply Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/MicrosoftManagementActivity.py:320]: one of the lints raised an error and I remembered it once said list() is better. Why is [] better? :)

Comment (orlichter1) [reply Packs/MicrosoftManagementActivity/Integrations/MicrosoftManagementActivity/MicrosoftManagementActivity.py:320]: https://stackoverflow.com/questions/5790860/and-vs-list-and-dict-which-is-better Found an answer

**Comment by kirbles19** [comment]
@orlichter1 Done.
